sql/moz-fx-data-shared-prod/contextual_services/adm_forecasting/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
```

Comment:
This view needs a metadata file that specifies access for `workgroup:contextual-services` group. See https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/contextual_services/event_aggregates/metadata.yaml

Reply:
Good call, added a metadata file in the regular folder and added the workgroup access to the one in the derived folder as well.